optuna/trial.py

```diff
@@ -559,6 +560,16 @@ def system_attrs(self):
 
         return self.storage.get_trial_system_attrs(self._trial_id)
 
+    @property
+    def datetime_start(self):
+        # type: () -> Optional[datetime]
+        """Return start datetime.
+
+        Returns:
+            Datetime where the :class:`~optuna.trial.Trial` started.
+        """
```

Comment:
To be consistent with most of other methods, please add an empty line after the docstring.

Reply:
We'll address this in a followup PR.